website/src/controllers/user.js

```diff
@@ -97,7 +97,9 @@ api.score = function(req, res, next) {
       id: id,
       type: req.body && req.body.type,
       text: req.body && req.body.text,
-      notes: (req.body && req.body.notes) || "This task was created by a third-party service. Feel free to edit, it won't harm the connection to that service. Additionally, multiple services may piggy-back off this task."
+      notes: (req.body && req.body.notes) || "This task was created by a third-party service. Feel free to edit, it won't harm the connection to that service. Additionally, multiple services may piggy-back off this task.",
```

Comment:
This, along with the the other strings on this file, should probably be translatable. This can be achieved by using the shared module, like this: `shared.i18n.t('stringName')` and adding new string variables to [the task.json file](https://github.com/HabitRPG/habitrpg/blob/develop/common/locales/en/tasks.json)